webapp/src/pages/ReponseDetails/ResponseDetails.css

```diff
@@ -0,0 +1,408 @@
+/* Universal */
```

Comment:
Is all of this additional CSS really needed? Also there seems to be a some very generally selectors like html button which should not belong inside a component-specific CSS file.